src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationAnalysisChartPresenter.cs

```diff
@@ -78,9 +79,10 @@ protected override void ConfigureColumns()
 
       private void showSimulationColumn()
       {
-         Column(BrowserColumns.Simulation).Visible = true;
-         Column(BrowserColumns.Simulation).VisibleIndex = 0;
-         ChartEditorPresenter.ApplyColumnSettings();
+         var simulationColumnSettings = Column(BrowserColumns.Simulation);
+         simulationColumnSettings.Visible = true;
+         simulationColumnSettings.VisibleIndex = 0;
+         ChartEditorPresenter.ApplyColumnSettings(simulationColumnSettings);
```

Comment:
At that stage, there might be another column with index 0. So applying ALL Settings would not work. Instead I need to apply only this one setting